sampleconfig/configtx.yaml

```diff
@@ -310,6 +310,14 @@ Orderer: &OrdererDefaults
     # network. When set to 0, this implies no maximum number of channels.
     MaxChannels: 0
 
+    ConsenterMapping:
+    - ID: 1
+      MSPID: OrdererOrg1
+      Identity: <base64 encoding of raw identity>
+    - ID: 2
+      MSPID: OrdererOrg2
+      Identity: <base64 encoding of raw identity>
```

Comment:
1. Let's put here 4 entries, since it's the minimum in BFT
2. Shall we put here the paths of the identitiy like we do in Raft?